resources/tentacle.rb

```diff
@@ -18,7 +18,7 @@
 # limitations under the License.
 #
 
-actions :install, :configure, :remove, :uninstall
+actions :install, :configure, :register, :remove, :uninstall
```

Comment:
Since we are adding the register action I think we should make the version not required so it doesn't need to be passed in.  Then validation should be added for the other methods to keep the same behavior

```
attribute :version, kind_of: String, required: true
```